automl/cloud-client/get_model_evaluation.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def get_model_evaluation(project_id, model_id, model_evaluation_id):
+    """Get model evaluation."""
+    # [START automl_language_entity_extraction_get_model_evaluation]
+    # [START automl_language_sentiment_analysis_get_model_evaluation]
+    # [START automl_language_text_classification_get_model_evaluation]
+    # [START automl_translate_get_model_evaluation]
+    # [START automl_vision_classification_get_model_evaluation]
+    # [START automl_vision_object_detection_get_model_evaluation]
+    from google.cloud import automl
+
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
```

Comment:
If you're going to change to double quotes elsewhere, you should also change these here.